cloudfoundry/resource_cf_app.go

```diff
@@ -254,9 +260,32 @@ func resourceApp() *schema.Resource {
 							Type:     schema.TypeString,
 							Optional: true,
 						},
-						"version": &schema.Schema{
```

Comment:
I simply removed this field due to not expecting it to be in use by anyone since it has never had any implemented functionality.  Also, it is not currently documented.